pkg/tasks/tasks.go

```diff
@@ -45,11 +45,12 @@ func (tl *TaskRunner) RunAll() error {
 			klog.V(3).Infof("running task %d of %d: %v", i+1, len(tl.tasks), ts.Name)
 			err := tl.ExecuteTask(ts)
 			klog.V(3).Infof("ran task %d of %d: %v", i+1, len(tl.tasks), ts.Name)
+			currTask = ts.Name
```

Comment:
unfortunately, this won't work, as writing to `currTask` is unsynchronized. Hence, `currTask` may end up empty string `""`, or any arbitrary task name.